### PR TITLE
Fix Safari publish problem

### PR DESCRIPTION
**Pull request by vpoddubchak**
After this [commit](https://github.com/lynckia/licode/pull/1659) Safari start to have problems with publishing.
As I understand, the same problem was with Firefox and it has been [fixed](https://github.com/lynckia/licode/pull/1669).

I did the same for Safari and it works in our environment.

### CODE REVIEW

File: erizo_controller/erizoClient/src/webrtc-stacks/SafariStack.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  const that = ChromeStableStack(specInput);
+
+  that._updateTracksToBeNegotiatedFromStream = () => {
+    that.tracksToBeNegotiated += 1;
+  };
+
+  return that;
+};
+
+export default SafariStack;
```
Comment (lodoyun) [inline erizo_controller/erizoClient/src/webrtc-stacks/SafariStack.js:16]: We need a new line at the end here

**Review by lodoyun — COMMENTED** [review]
Good catch. I think this is a good solution 👍.
Please fix the linter issues. I think it's only the one I commented.
Thanks!

File: erizo_controller/erizoClient/src/webrtc-stacks/SafariStack.js (hunk 1 above)
Comment (vpoddubchak) [reply erizo_controller/erizoClient/src/webrtc-stacks/SafariStack.js]: Done

**Review by lodoyun — APPROVED** [review]
👍

**Pull request merged**